### PR TITLE
Change default SDK to nightly in PR build

### PR DESCRIPTION
**Pull request by xfxcwynlc**
Change from releases to nightly at line 36, 73.
Closes: #2187 
Signed-off-by: Yanan Wang <xfxcwynlc@gmail.com>

### CODE REVIEW

**Comment by sophia-guo** [comment]
Is this duplicate or part of #2201 ?

**Comment by xfxcwynlc** [comment]
> Is this duplicate or part of #2201 ?

I created a new pull request and only focused on changing 'releases' to 'nightly', while #2201 is trying to fix 2 issues at the same time, one for ready_to_review and the second for 'releases to nightly', but the later one forgot to change line 73.

**Review by llxia — APPROVED** [review]
LGTM

File: .github/workflows/PushOrPRtestbuild.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
     - uses: AdoptOpenJDK/install-jdk@v1
       with:
         version: ${{ matrix.version }}
-        source: 'releases'  #nightly for nightly test builds
+        source: 'nightly'  #nightly for nightly test builds
```
Comment (sophia-guo) [inline .github/workflows/PushOrPRtestbuild.yml:36]: Might just remove the comment as it doesn't help much.

File: .github/workflows/PushOrPRtestbuild.yml [hunk 2]
```diff
@@ -70,7 +70,7 @@ jobs:
...
```
Comment (sophia-guo) [inline .github/workflows/PushOrPRtestbuild.yml:73]: Same as comment above, just remove the comment.

**Review by sophia-guo — CHANGES_REQUESTED** [review]

**Comment by smlambert** [comment]
@xfxcwynlc if you remove the unneeded comments as @sophia-guo requested then this PR will be ready to merge.

**Review by sophia-guo — APPROVED** [review]
LGTM

**Pull request merged**